docs/admin/multiple-zones.md

```diff
@@ -0,0 +1,280 @@
+---
+---
+
+## Introduction
+
+Kubernetes 1.2 adds support for running a single cluster in multiple zones.
+This is a lightweight version of a broader effort for federating multiple
+Kubernetes clusters together (sometimes referred to by the affectionate
```

Comment:
Good to add a link to further information here.
For now, I suggest that you use https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/federation.md
In a separate PR I will put together an index page linking to all of the Ubernetes docs, and update the link in this doc.